crates/egui/src/ui.rs

```diff
@@ -310,8 +316,13 @@ impl Ui {
 
     /// Use this to paint stuff within this [`Ui`].
     #[inline]
-    pub fn layer_id(&self) -> LayerId {
-        self.painter().layer_id()
+    pub fn layer_id(&self) -> AreaLayerId {
+        self.painter().layer()
+    }
+
+    #[inline]
+    pub fn layer(&self) -> ZLayer {
+        self.painter().zlayer()
```

Comment:
```suggestion
        self.painter().z_layer()
```

or I will pronounce it as "slayer" :)